test/persistence-hooks.test.js

```diff
@@ -14,5 +14,10 @@ if (process.env.MONGODB_VERSION &&
 for(var i in config) {
   customConfig[i] = config[i];
 }
+var DB_VERSION = process.env.MONGODB_VERSION;
+var DB_HAS_2_6_FEATURES = (!DB_VERSION ||
```

Comment:
@bajtos: Please note: if you do `npm test` locally we assume you have mongodb 2.6+ and supports `isNewInstance`